packages/gatsby-mdx/utils/gen-mdx.js

```diff
@@ -87,27 +79,13 @@ module.exports = async function genMDX({
    *   reporter,
    *   cache
    * }); */
-
-  const gatsbyRemarkPluginsAsMDPlugins = await getSourcePluginsAsRemarkPlugins({
-    gatsbyRemarkPlugins: options.gatsbyRemarkPlugins,
-    mdxNode: node,
-    //          files,
-    getNode,
-    getNodes,
-    reporter,
-    cache,
-    pathPrefix
-  });
-
   debug("running mdx");
-  let code = await mdx(content, {
-    ...options,
-    mdPlugins: options.mdPlugins.concat(gatsbyRemarkPluginsAsMDPlugins)
-  });
+  const code = await mdx(content, options);
 
   debug("compiling scope");
   const instance = new BabelPluginPluckImports();
   const result = babel.transform(code, {
+    ...babelConfig,
```

Comment:
I don't see a compelling reason to make this configurable yet (unless you had a use case in mind?). Let's add `configFile: false` directly here instead.